native/cocos/platform/android/java/src/com/cocos/lib/CocosEditBoxActivity.java

```diff
@@ -466,7 +466,7 @@ public void run() {
      Native functions invoked by UI.
      **************************************************************************************/
     private void onKeyboardInput(String text) {
-        CocosHelper.runOnGameThreadAtForeground(new Runnable() {
+        CocosHelper.runOnGameThread(new Runnable() {
```

Comment:
What's the difference between these two functions?

Reply:
When an activity is in the foreground (activity is in the resume state), two interfaces can be called, but when the activity is in the pause state, only runOnGameThread interface can be called

Reply:
Got it. But if the activity in pause state, why `onKeyboardInput ` can be invoked?

Reply:
CocosEditBoxActivity is a transparent activity, game can be show normally, and game's render loop still running

Reply:
Got it.